integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CarbonScalaUtil.scala

```diff
@@ -70,7 +70,7 @@ object CarbonScalaUtil {
       level: Int = 0): String = {
     FieldConverter.objectToString(value, serializationNullFormat, complexDelimiters,
       timeStampFormat, dateFormat, isVarcharType = isVarcharType, isComplexType = isComplexType,
-      level)
+      level, true)
```

Comment:
this method is an static method in Util, why the last input paramer 'insertflow' always use 'true' ?

Reply:
Currently, the method is only invoked by insert flow, so  the insertflow parameter is fixed with 'true'.